api/config/routes.rb

```diff
@@ -281,11 +281,11 @@
           resources :teams, only: [:index]
 
           if Settings&.dashboard&.marketplace&.provisioning
-            resources :subscriptions, only: [:index, :show, :create, :update] do
-              member do
-                post :cancel
-              end
+            resources :subscription_events, only: [:index]
+          end
 
+          if Settings&.dashboard&.marketplace&.provisioning
```

Comment:
You can group this within the same `if Settings&.dashboard&.marketplace&.provisioning` above

Reply:
lol oops thanks.